airflow/www/decorators.py

```diff
@@ -39,17 +39,19 @@ def wrapper(*args, **kwargs):
             else:
                 user = g.user.username
 
+            params = request.form if request.method == 'POST' else request.args
+
             log = Log(
                 event=f.__name__,
                 task_instance=None,
                 owner=user,
-                extra=str(list(request.args.items())),
-                task_id=request.args.get('task_id'),
-                dag_id=request.args.get('dag_id'))
+                extra=str(list(params.items())),
```

Comment:
Just use [request.values](https://werkzeug.readthedocs.io/en/0.15.x/wrappers/#werkzeug.wrappers.BaseRequest.values) instead.

```suggestion
                extra=str(list(request.values.items())),
```

Reply:
Though I do worry that this will capture passwords and other sensitive info from the Variables form etc.

Reply:
Thanks for the suggestion. I updated the code.

Regarding your other concern: `this will capture passwords and other sensitive info`
I don't have a great way of addressing that. So I did this unscientific search in the code base. @action_logging decorator is not used on the ConnectionModelView. So sensitive things put in the form is not logged.

`$ grep -A5 action_logging airflow/www/views.py`

I also started a test server and entered some info in ConnectionModelView and checked nothing is logged. 
That said, pls do let me know if you have better ways to address or verify this.